source/client-side-encryption/tests/delete.template.yml

```diff
@@ -0,0 +1,58 @@
+runOn:
```

Comment:
Test both deleteOne and deleteMany (as you already do with update)

Reply:
Done.